### PR TITLE
Implement the new ModelInfo method in the sandbox

### PR DESCRIPTION
**Pull request by huwshimi**
This is to stop the warning when the GUI is loaded in sandbox mode.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the GUI in sandbox mode
- there should be no error in your browser console about a missing `handleModelManagerModelInfo` method

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5626/
Test PASSed.

**Comment by anthonydillon** [comment]
Looks good thanks :+1:

File: jujugui/static/gui/src/app/store/env/sandbox.js [hunk 1]
```diff
@@ -337,6 +337,34 @@ YUI.add('juju-env-sandbox', function(Y) {
...
+    @return {undefined} Side effects only.
+    */
+    handleModelManagerModelInfo: function(data, client, state) {
+      client.receive({
+        'request-id': data['request-id'],
+        response: {
+          results: [{
+            result: {
+              name: 'sandbox',
+              'default-series': 'trusty',
```
Comment (frankban) [inline jujugui/static/gui/src/app/store/env/sandbox.js]: I think we should use state.get('providerType') and state.get('defaultSeries') to get the values here, for consistency.

**Comment by frankban** [comment]
Thanks for this Huw!
👍  with a trivial suggestion and after adding at least a test for the functionality. It can be similar to the existing one for Client.ModelInfo (a call that no longer exists but that I will restore later): see it('can return model information.'.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5627/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @anthonydillon, @frankban :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**